src/js/tech/flash-rtmp.js

```diff
@@ -104,8 +104,7 @@ function FlashRtmpDecorator(Flash) {
   /**
    * Regular expression used to check if the source is an rtmp source.
    *
-   * @property
-   * @type {RegExp}
+   * @property {RegExp} Flash.RTMP_RE
```

Comment:
I believe this could simply be `@type {RegExp}` without the `@property` entirely.